src/teleport.c

```diff
@@ -168,11 +167,6 @@ void spawn_braineaters(float chance)
     }
 }
 
-plane *get_normalplane(void)
-{
-    return NULL;
-}
-
```

Comment:
Es gehört auf jeden Fall nicht in teleport.c, und eine Funktion die nur NULL zurückliefert, ist auch ein bisschen überflüssig, fand ich. Die Normalebene ist durch region.plane == NULL indiziert, und hat kein eigenes Plane-Objekt.